.gitignore

```diff
@@ -108,3 +108,6 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+# Pycharm
```

Comment:
Not using vim 😢

Reply:
@george-ayris has the skills, just not the inclination

Reply:
Need to get my vim setup at work...